Code/CoreData/RKManagedObjectStore.h

```diff
@@ -24,6 +24,8 @@
 
 @class RKManagedObjectStore;
 
+extern NSSet *RKSetOfManagedObjectIDsFromManagedObjectContextDidSaveNotification(NSNotification *notification);
```

Comment:
I'd just redeclare this in the test file, I don't think we should be exposing this as part of public API